clients/docs/ksql-datagen.rst

```diff
@@ -99,7 +99,6 @@ Consume Records
 
       .. code-block:: bash
 
-         docker-compose exec connect bash -c 'kafka-console-consumer --topic test1 --bootstrap-server $CONNECT_BOOTSTRAP_SERVERS --consumer-property ssl.endpoint.identification.algorithm=https --consumer-property sasl.mechanism=PLAIN --consumer-property security.protocol=SASL_SSL --consumer-property sasl.jaas.config="$SASL_JAAS_CONFIG_PROPERTY_FORMAT" --max-messages 5'
```

Comment:
@ybyzek it's difficult to see how this removal is related to DEVX-2151, but if it's being removed, should the code-block directive above it be removed as well?

Reply:
My script was too aggressive in deleting lines...fixing now

Reply:
And good catch ;)